src/components/Header/AddEditDialog/AddEditDialog.tsx

```diff
@@ -33,78 +51,87 @@ export const AddEditDialog = (props: AddEditDialogProps): JSX.Element => {
-              type="text"
-              value=""
-              placeholder="7.8"
-            />
-          </div>
-          <div className="mwFormRow">
-            <MwInput
-              className="mwMiddleInput"
-              containerStyle=""
-              name="Genre"
-              onChange={onTitleChange}
-              type="text"
-              value=""
-            />
-            <MwInput
-              className="mwShortInput"
-              containerStyle=""
-              name="Runtime"
-              onChange={onRuntimeChange}
-              type="text"
-              value=""
-              placeholder="minutes"
-            />
-          </div>
-          <div className="mwFormRow">
-            <MwInput
-              className="mwLongInput"
-              containerStyle=""
-              name="Overview"
-              onChange={onRuntimeChange}
-              type="textarea"
-              value=""
-              placeholder="movie description"
-            />
-          </div>
+          <form onSubmit={onSubmitClick}>
+            <div className="mwFormRow">
+              <MwInput
+                id="title"
+                className="mwMiddleInput"
+                containerStyle=""
+                name="title"
+                onChange={handleInputChange}
+                type="text"
+                value={movieItem?.title}
+              />
+              <MwInput
+                id="releaseDate"
+                className="mwShortInput"
+                containerStyle=""
+                name="Release date"
+                onChange={handleInputChange}
+                type="date"
+                value={movieItem?.releaseDate}
+              />
+            </div>
+            <div className="mwFormRow">
+              <MwInput
+                id="posterPath"
+                className="mwMiddleInput"
+                containerStyle=""
+                name="Movie URL"
+                onChange={handleInputChange}
+                type="text"
+                value={movieItem?.posterPath}
+                placeholder="https://"
+              />
+              <MwInput
+                id="voteAverage"
+                className="mwShortInput"
+                containerStyle=""
+                name="Rating"
+                onChange={handleInputChange}
+                type="string"
+                value={movieItem?.voteAverage.toString()}
+                placeholder="7.8"
+              />
+            </div>
+            <div className="mwFormRow">
+              <MwInput
+                id="genres"
+                className="mwMiddleInput"
+                containerStyle=""
+                name="Genre"
+                onChange={handleInputChange}
+                type="text"
+                value={movieItem?.genres.toString()}
+              />
+              <MwInput
+                id="runtime"
+                className="mwShortInput"
+                containerStyle=""
```

Comment:
Is this prop even used in input? Having an empty value assigned to each input is not a good idea, I think.

Reply:
yes, when I first add it, I think there was an intension to use it, then I forgot about it.